controller/AbstractRestQti.php

```diff
@@ -219,7 +221,7 @@ public function updateMetadata(): void
             throw new BadRequestException($exception->getMessage(), 422);
         }
 
-        $this->returnSuccess($metadataInput);
+        $this->setSuccessJsonResponse($metadataInput);
```

Comment:
Ok I found one thing, its abstract class so possible many extensions of it. Are we sure in every place we want to continue with those event you @andreluizmachado showed us on meeting? Maybe somewhere is some piece of code whitch depends on this removed exit() ? Can we somehow check it?

Reply:
Very good Question @Karol-Stelmaczonek. But it should be ok, actually, this implementation is quite new and for sure that events are mandatory to be thrown anywhere.